### PR TITLE
Fix/daef 438 Fix logger statement for the api call for calculating transaction fee

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR introduces a fix for logger statement in the API call for calculating transaction fee.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Changelog
 - Fixed ENOENT error on log file rotation ([PR 428](https://github.com/input-output-hk/daedalus/pull/428))
 - Fixed logging error ([PR 431](https://github.com/input-output-hk/daedalus/pull/431))
 - Add theming support on UI which was introduced after introduction of Theming feature ([PR 434](https://github.com/input-output-hk/daedalus/pull/434))
+- Fixed logger statement for the api call for calculating transaction fee ([PR 442](https://github.com/input-output-hk/daedalus/pull/442))
```
Comment (darko-mijic) [inline CHANGELOG.md]: I don't think we need minor details as this one in the changelog.

Comment (nikolaglumac) [reply CHANGELOG.md]: Agreed! ...and removed ;)

**Comment by darko-mijic** [comment]
@domenkozar: This is ready to be merged but the CI is failing.

**Comment by domenkozar** [comment]
Yeah, you merged a failing CI in #432 - but it will get fixed with #443 once it's ready

**Comment by darko-mijic** [comment]
Closing this PR since changes introduced here were already been done is the scope of other PRs.

**Pull request closed without merging**